osu.Game.Rulesets.Mania/Mods/ManiaModNoLongNotes.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Linq;
+using osu.Framework.Extensions.IEnumerableExtensions;
+using osu.Game.Beatmaps;
+using osu.Game.Rulesets.Mania.Objects;
+using osu.Game.Rulesets.Mods;
+using osu.Framework.Graphics.Sprites;
+using System;
+using System.Collections.Generic;
+using osu.Game.Audio;
+using osu.Game.Rulesets.Mania.Beatmaps;
+using osu.Framework.Utils;
+using osu.Game.Overlays.Settings;
+using osu.Framework.Bindables;
+using osu.Game.Configuration;
+using osu.Game.Graphics;
+
+
+
+namespace osu.Game.Rulesets.Mania.Mods
+{
+    public class ManiaModNoLongNotes : Mod, IApplicableAfterBeatmapConversion
+    {
+
+        public override string Name => "No Long Notes";
```

Comment:
Name feels a bit too long, maybe "Taps Only" or something?

Reply:
not sure "taps" fits with the rest of the mania vocabulary. the notes are technically called "hold" notes most everywhere else so that is probably preferred. you could even call this the "No Holds" mod.

Reply:
fwiw in osu!mania the term for this is "rice" so if any i'd like to suggest `Rice Only` or sth like that

![image](https://user-images.githubusercontent.com/36564236/151410440-316371d0-0a2a-4229-ba86-2a261abfd0e2.png)

Reply:
"rice" is a colloquial / community term, and i personally found it was very confusing when i first encountered it. i don't see any references to that term [on the wiki](https://osu.ppy.sh/home/search?mode=wiki_page&query=rice) either, outside of tournament articles.

Reply:
seems like we'll need to write stubs on these and give the community terms a mention there

![image](https://user-images.githubusercontent.com/36758269/151411371-3d527d03-e241-469a-9e15-d7d7cdb14d6f.png)

Reply:
Rice Only does have a nice ring to it. 

> "rice" is a colloquial / community term, and i personally found it was very confusing when i first encountered it. 
With a proper description, it might be possible to avoid any confusion, if the mod clearly states what it means. That said, maybe the name is still a bit too colloquial? No Holds is a solid improvement. I would change it to that if people agree

Reply:
I’m okay with witty names but let’s not use “rice”.